packages/grafana-ui/src/components/Button/ButtonGroup.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { forwardRef, HTMLAttributes } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useStyles } from '../../themes';
+
+export interface Props extends HTMLAttributes<HTMLDivElement> {
+  noSpacing?: boolean;
+}
+
+export const ButtonGroup = forwardRef<HTMLDivElement, Props>(({ noSpacing, children, ...rest }, ref) => {
+  const styles = useStyles(getStyles);
+  const className = cx({
+    [styles.wrapper]: !noSpacing,
+    [styles.wrapperNoSpacing]: noSpacing,
+  });
```

Comment:
```suggestion
  const className = noSpacing ? styles.wrapperNoSpacing : styles.wrapper;
```
Not sure we need `cx` here, the styles seem to be mutually exclusive so we can just pick one based on condition.

Reply:
ah, great spotted!